doc/modules/feature_extraction.rst

```diff
@@ -916,6 +916,30 @@ Some tips and tricks:
 
     (Note that this will not filter out punctuation.)
 
+
+    The following example will, for instance, transform British spelling to American
```

Comment:
This is when I say "so much for all those input parameters; overloading ftw!"

I'm sure this kind of thing is done in the wild, so we might as well advertise it.

LGTM!